Podcast/BookmarksTableViewCell.swift

```diff
@@ -0,0 +1,228 @@
+        recommendedButton = UIButton(frame: CGRect.zero)
+        recommendedButton.setImage(#imageLiteral(resourceName: "bookmarks_recommend_small"), for: .normal)
+        recommendedButton.setImage(#imageLiteral(resourceName: "bookmarks_recommend_small_selected"), for: .selected)
+        recommendedButton.contentHorizontalAlignment = .left
+        recommendedButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: buttonTitlePadding, bottom: 0, right: 0)
+        recommendedButton.setTitleColor(.podcastGrayDark, for: .normal)
+        recommendedButton.setTitleColor(.cancelButtonRed, for: .selected)
+        recommendedButton.setTitle("0", for: .normal)
+        recommendedButton.titleLabel?.font = .systemFont(ofSize: 12)
+        recommendedButton.addTarget(self, action: #selector(didPressRecommendedButton), for: .touchUpInside)
+        addSubview(recommendedButton)
+        
+        moreButton = UIButton(frame: CGRect.zero)
+        moreButton.setImage(#imageLiteral(resourceName: "more_icon"), for: .normal)
+        moreButton.addTarget(self, action: #selector(didPressMoreButton), for: .touchUpInside)
+        addSubview(moreButton)
+        
+        heightConstraint = NSLayoutConstraint(item: contentView, attribute: .height, relatedBy: .equal, toItem: nil, attribute: NSLayoutAttribute.notAnAttribute, multiplier: 1, constant: height)
+        contentView.addConstraint(heightConstraint!)
+        NSLayoutConstraint.activate([heightConstraint])
+    }
+    
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        episodeNameLabel.frame = CGRect(x: episodeNameLabelX, y: episodeNameLabelY, width: frame.width - padding - episodeNameLabelX, height: episodeNameLabelHeight)
+        dateTimeLabel.frame = CGRect(x: dateTimeLabelX, y: dateTimeLabelY, width: frame.width - dateTimeLabelX - padding, height: dateTimeLabelHeight)
+        episodeImage.frame = CGRect(x: padding, y: padding, width: episodeImageSideLength, height: episodeImageSideLength)
+        
+        playButton.frame = CGRect(x: playButtonX, y: frame.height - playButtonBottomY - playButtonHeight, width: playButtonWidth, height: playButtonHeight)
+        recommendedButton.frame = CGRect(x: recommendedButtonX, y: frame.height - recommendedButtonBottomY - recommendedButtonHeight, width: recommendedButtonWidth, height: recommendedButtonHeight)
+        moreButton.frame = CGRect(x: frame.width - moreButtonWidth - moreButtonRightX, y: frame.height - moreButtonBottomY - moreButtonHeight, width: moreButtonWidth, height: moreButtonHeight)
+        
+        separator.frame = CGRect(x: 0, y: frame.height - separatorHeight, width: frame.width, height: separatorHeight)
+    }
+    
+    func setupWithEpisode(episode: Episode) {
+        
+        episodeNameLabel.text = episode.title
+        
+        let dateFormatter = DateFormatter()
```

Comment:
I can make this into it. I think when first written in Card cell we didn't know how many times it would be used, now it's clear it's everywhere haha.